docs/srap.csv

```diff
@@ -36,7 +36,8 @@ Scholarly Resource,Type,dct:type,IRI,,,http://purl.org/coar/resource_type/,IRIst
 ,,,,,,,,
 ,Accessibility statement,srap:accessibilityStatement,literal,xsd:string,,,,"Textual information describing the accessibility features of a resource, including technical details."
 ,,,,,,,,
-,Grant number,srap:grantNumber,literal,xsd:string,,,,"An alpha-numeric string identifying the contract, project or funding grant under which the scholarly resource was created."
+,Grant,schema:funding,,,Grant,,,The funder of this resource.
```

Comment:
I think this should be "The funding of this resource" because this is not a direct link to the funder (organization)

Reply:
Yes, "funding" is better. Feel free to make the change.